src/styles/blocks/card.scss

```
@@ -0,0 +1,99 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  width: 950px;
```

you can use gap instead margin in cards
```suggestion
  flex-wrap: wrap;
  column-gap: 48px;
  row-gap: 46px;
  width: 950px;
```

---

flex Gap does not work for our tests.